### PR TITLE
Add rector rules for CakePHP 4.3

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

**Comment by samsonasik** [comment]
Please run `composer fix-cs` to fix cs .

File: config/sets/cakephp43.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                new PropertyFetchToMethodCall('Cake\Network\Socket', 'connected', 'isConnected'),
+                new PropertyFetchToMethodCall('Cake\Network\Socket', 'encrypted', 'isEncrypted'),
+                new PropertyFetchToMethodCall('Cake\Network\Socket', 'lastError', 'lastError'),
+            ]),
+        ]]);
+
+    $services->set(MethodCallToAnotherMethodCallWithArgumentsRector::class)
+        ->call('configure', [[
+            MethodCallToAnotherMethodCallWithArgumentsRector::METHOD_CALL_RENAMES_WITH_ADDED_ARGUMENTS => ValueObjectInliner::inline([
+
```
Comment (markstory) [inline config/sets/cakephp43.php:37]: @samsonasik I've run `composer fix-cs` and it added this newline with trailing whitespace. I don't know if that's what should be done but on subsequent runs locally this line is changed each time.

Comment (markstory) [reply config/sets/cakephp43.php]: Figured it out the line length fixer was misbehaving.

**Comment by samsonasik** [comment]
Thank you @markstory

**Review by samsonasik — APPROVED** [review]

**Pull request merged**